### PR TITLE
Replace magic auth scheme strings

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/Docs/issues/4625

[Internal Review Page](https://review.docs.microsoft.com/aspnet/core/security/authorization/limitingidentitybyscheme?branch=pr-en-us-4626)

### CODE REVIEW

**Comment by HaoK** [comment]
I don't believe so no, this was how Roles worked in the past so we kept that syntax (comma separated)

**Comment by HaoK** [comment]
Note, if the defaults are ugly and long, you can always specify the scheme explicitly in AddCookie/AddJwtBearer and use the schemes in Authorize.

**Comment by scottaddie** [comment]
@HaoK My opinion is that we should encourage using the constants. They're in the framework, after all. Are you and @Tratcher okay with that approach?

**Comment by Tratcher** [comment]
Constants are good, but you may want to make a local constant in the file for combining the two.

**Comment by scottaddie** [comment]
@Tratcher Please take a look at the latest changes. Is this what you had in mind?

File: aspnetcore/security/authorization/limitingidentitybyscheme.md [hunk 1]
```diff
@@ -82,15 +82,27 @@ At the point of authorization, the app indicates the handler to be used. Select
 # [ASP.NET Core 2.x](#tab/aspnetcore2x)
 
 ```csharp
-[Authorize(AuthenticationSchemes = "Cookie,Bearer")]
+[Authorize(AuthenticationSchemes = AuthSchemes)]
```
Comment (Tratcher) [inline aspnetcore/security/authorization/limitingidentitybyscheme.md]: Can an attribute on a class reference a (private) constant defined in that class?

Comment (scottaddie) [reply aspnetcore/security/authorization/limitingidentitybyscheme.md]: I didn't think it was possible either, but it does compile.

Comment (Tratcher) [reply aspnetcore/security/authorization/limitingidentitybyscheme.md]: Wow. 👍

**Pull request merged**